kf_hungarian_tracker/README.md

```diff
@@ -0,0 +1,27 @@
+# kf_hungarian_tracker
+
+This package implememts a multi-object tracker with Kalman filter and Hungarian algorithm. 
+Hungarian algorithm is used to associate detection and previously known objects. 
+For each object, a Kalman filter is maintained to track the position and velocity. 
+
+### Hungarian algorithm
+
+[Hungarian algorithm](https://en.wikipedia.org/wiki/Hungarian_algorithm) is a combinatorial optimization algorithm that solves the assignment problem in polynomial time. 
+Here we use an implementation from `scipy.optimize.linear_sum_assignment`. The default cost function is Euclidean distance between two object's center. 
+More cost functions like IoU of bounding boxes could be used.
+
+### Parameters
+
+Parameters can be set in `config/kf_hungarian.yaml`. For more information on parameters for Kalman filter, check out [KalmanFilter](https://docs.opencv.org/master/dd/d6a/classcv_1_1KalmanFilter.html) from OpenCV. 
+
+| parameters       | Meaning        | Default |
+| ---------------- | ------------- | ------- |
+| global_frame     | transform from pointcloud frame to global frame,<br>None means message frame is global  | camera_link   |
+| death_threshold  | maximum missing frames before deleting an obstacle  | 3 |
+| top_down | whether project 3D points on ground plane, set to do top-down tracking | False |
+| measurementNoiseCov | measurement noise for Kalman filter [x,y,z] | [1., 1., 1.] |
+| errorCovPost | initial posteriori error estimate covariance matrix [x,y,z,vx,vy,vz] | [1., 1., 1., 10., 10., 10.] |
+| a_noise | model process noise covariance with estimated acceleration [ax,ay,az] | [2., 2., 0.5] |
```

Comment:
should `a_noise` be in `errorCovPost`? 

Also, your naming is really varying. Most are snake_case, but a couple of these are CamelCase (errorCovPost, ameasurementNoiseCov), can you have them all snake_case?